osquery/tables/system/windows/registry.cpp

```diff
@@ -72,6 +72,21 @@ void explodeRegistryPath(const std::string& path,
   rKey = osquery::join(toks, kRegSep);
 }
 
+Status sanitizeRegistryStrings(const DWORD dataType,
+                               BYTE* dataBuff,
+                               DWORD dataSize) {
+  auto status = Status(0, "OK");
+  if (dataBuff != NULL && dataSize != NULL) {
```

Comment:
I know that the `NULL`s exist throughout our code base, but we're trying to encourage folks to use `nullptr` instead. Would you mind swapping these out?